flake.nix

```diff
@@ -211,6 +211,8 @@
           profiling-script = pkgs.callPackage ./nix/run-profiling.nix {
             inherit (pkgs.haskellPackages)
               hp2pretty hs-speedscope eventlog2html;
+            kore-exec =
```

Comment:
Question about how nix works here: will this create a separate, normal, build of `kore-exec`?

Reply:
yes, `self.project` referred to here is defined as:

https://github.com/runtimeverification/haskell-backend/blob/ebfd83c08b353177cc1f6b9168d8f5412ac2dae1/flake.nix#L154-L160

it defaults to `profiling = false;`, unlike `projectProfilingEventlog` underneath.